pkgs/development/tools/parsing/tree-sitter/update_impl.py

```diff
@@ -17,6 +18,15 @@
 Args = Iterator[str]
 
 
+# Latest release from neovim https://api.github.com/repos/tree-sitter/tree-sitter-bash/releases/latest
+# TODO: Unhardcode this? Pass it in from nix?
+MIN_TIMESTAMP = "2023-05-29T11:24:38Z"
```

Comment:
Apologies for hardcoding this, I just didn't have time to refactor the arg passing right now.

Reply:
It should be committed: I think it could be important to keep track of the date used to update the grammars. Theoretically the commit hashes should be enough, but such timestamp could be more easy to reason about.

Said so, maybe it is better to move it to [`default.nix`](https://github.com/NixOS/nixpkgs/blob/a1240ab9f02649dd48ce0ba10ab158e0061a04c5/pkgs/development/tools/parsing/tree-sitter/default.nix#L20) (with a proper comment), along side with other tree-sitter update variables.

Reply:
That would be ideal! Would you be interested in doing this refactor? I'm limited on further work on this at the moment. :)

Reply:
Yes I could do it in the next days